stable/contour/README.md

```diff
@@ -36,7 +36,8 @@ The default configuration values for this chart are listed in `values.yaml`.
 | `contour.image.pullPolicy`           | Image pull policy                                                     | `IfNotPresent`                             |
 | `contour.replicas`                   | Replica count for the contour deployment                              | `2`                                        |
 | `contour.resources`                   | 	Resource definitions for the contour pods                        | `{}`                                       |
-| `customResourceDefinitions.create`   | Wether the release should install CRDs                                | `true`                                     |
+| `customResourceDefinitions.create`   | Wether the release should install CRDs. Regardless of this value, Helm v3+ will install the CRDs if those are not present already. Use `--skip-crds` with `helm install` if you want to skip CRD creation                               | `true`
```

Comment:
Tiny spelling fix:

```suggestion
| `customResourceDefinitions.create`   | Whether the release should install CRDs. Regardless of this value, Helm v3+ will install the CRDs if those are not present already. Use `--skip-crds` with `helm install` if you want to skip CRD creation                               | `true`
```

Also, this is an excellent note 💯 All the relevant charts in #19008 should probably also get some variation of this note (though tons of readme's list explicit Helm commands for Helm 2, which also need to change. I think this will be helpful for people to understand how to navigate the CRDs though).